signer-cloud-server/src/main/java/com/wultra/signercloud/server/signer/SignerController.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Signer Cloud
+ * Copyright (C) 2025 Wultra s.r.o.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.wultra.signercloud.server.signer;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import jakarta.validation.Valid;
+import lombok.AllArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+/**
+ * Controller for {@link Signer} operations.
+ *
+ * @author Michal Rozehnal, michal.rozehnal@wultra.com
+ */
+@RestController
+@RequestMapping("api/signers")
+@AllArgsConstructor
+public class SignerController {
+
+    private final SignerService signerService;
+
+    @Operation(
+            summary = "Create a new signer",
+            description = "Creates a new signer with provided data.",
+            responses = {
+                    @ApiResponse(
+                            responseCode = "200",
+                            description = "REST API call was successful. Check the 'result' field in the response to determine if the signer was actually created.",
+                            content = @Content(schema = @Schema(implementation = SignerResponse.class))
+                    ),
+                    @ApiResponse(
+                            responseCode = "400",
+                            description = "Invalid input data"
+                    )
+            }
+    )
+    @PostMapping
+    public ResponseEntity<SignerResponse> create(@Valid @RequestBody CreateSignerRequest createSignerRequest) {
```

Comment:
Nitpick: I am just obsessed by `final`
```suggestion
    public ResponseEntity<SignerResponse> create(@Valid @RequestBody final CreateSignerRequest createSignerRequest) {
```

Reply:
Added